requirements/prod.txt

```diff
@@ -110,7 +110,7 @@ statsd==2.0.3
 suds==0.4
 tower==0.4.1
 # urllib3 is required by elasticsearch
-urllib3==1.8.2
+urllib3==1.10.3
```

Comment:
It's actually not ; requests ships with its own vendored version of urllib3, I updated it because I was in the neighbourhood :)